workers/ohsome_quality_analyst/cli.py

```diff
@@ -165,9 +163,25 @@ def create_indicator(
                 force=force,
             )
         )
-        # TODO: Save as GeoJSON instead of printing to Stdout
+        if outfile:
```

Comment:
Hey,
to be honest I'm a bit confused by the many if else statements in this function. Let's think about a way to make this eaier to read. Could you try to explain again what the major things are that you try to cover here and maybe this will help us to reduce the complexity of the code.

For instance here you check check `if outfile:` and a few lines later in line 179 you check it again.